src/main/kotlin/nevam/ReleaseCommand.kt

```diff
@@ -36,9 +37,19 @@ class ReleaseCommand : CliktCommand(name = "release") {
       .convert { MavenCoordinates.from(it) }
       .defaultLazy { MavenCoordinates.readFrom("gradle.properties") }
 
+  private val usernameProperty by option(
+      "-u", "--username",
+      help = "Global Gradle property defining the Sonatype Nexus username"
+  ).default(NexusUser.DEFAULT_USERNAME_PROPERTY)
+
+  private val passwordProperty by option(
+      "-p", "--password",
+      help = "Global Gradle property defining the Sonatype Nexus password"
```

Comment:
> I also feel like -u and -p are a bit misleading since it's looking for property names instead of actual values? But command line convention seems to be single-letter options.

Uhh yeah I agree and I can see this becoming confusing because `--password` points to another variable instead of the actual value. I'd suggest renaming this to something along the lines of `--password-property` and `-pp`.

OR what do you think of accepting both actual values and gradle properties? If the value starts with a `$` then startship reads it from `gradle.properties` otherwise treats it as the actual value?

Reply:
Oooh yeah I like that. Except, passwords could start with `$` so it would have to first try it as a property and then if that fails try it as a password.

And if it has to do both then it may as well not care about the `$` and just try it as both always.

Reply:
Oops, pushed before I saw this. I'm treating it as a key first and falling back to treating it as a value if it's not a key, regardless of prefix. Is the `$` prefix necessary?

Reply:
This looks good enough, let's go ahead with it!